test/EVMHost.h

```diff
@@ -127,6 +127,8 @@ class EVMHost: public evmc::MockedHost
 	static evmc::Result resultWithGas(int64_t gas_limit, int64_t gas_required, bytes const& _data) noexcept;
 	static evmc::Result resultWithFailure() noexcept;
 
+	static const std::array<evmc::bytes32, 2> m_blob_hashes;
```

Comment:
We don't generally use the `m_` prefix for static members. Also, I'd put such members last, to clearly visually separate them from all the static functions. Otherwise it's easy not to even notice that it's different.

```suggestion
	static std::array<evmc::bytes32, 2> const blobHashes;
```

Reply:
Also, I see you use this only once, in the constructor, to initialize the context. Why not just put the hard-coded values there along with all the other hard-coded constants?

Reply:
Indeed, I changed it to what you suggested. My initial thought was that it could be used in the `EVMInstructionInterpreter` as well, but it turned out that it doesn't.